### PR TITLE
docs: add css-minimizer-webpack-plugin to official plugins list

### LINKED ISSUE #3883: Added css-minimizer-webpack-plugin to official plugins
__Feature to document__

Need to add `css-minimizer-webpack-plugin` to official plugins https://github.com/webpack-contrib/css-minimizer-webpack-plugin

__Author(s)__
Any

__Additional information__

No

[ ] I am willing to work on this issue and submit a pull request.

### PR DESCRIPTION
**Pull request by jamesgeorge007**
closes #3883

### CODE REVIEW

File: src/components/Organization/projects.json [hunk 1]
```diff
@@ -250,5 +250,11 @@
 		"npm": "angular2-template-loader",
 		"description": "Load angular2 components, inlining all html and styling.",
 		"maintainer": "thelarkinn"
+	},
+	{
+		"repo": "webpack-contrib/css-minimizer-webpack-plugin",
+		"npm": "css-minimizer-webpack-plugin",
+		"description": "This plugin uses cssnano to optimize and minify your CSS.",
```
Comment (chenxsan) [inline src/components/Organization/projects.json:257]:
```suggestion
		"description": "Optimize and minify your CSS using cssnano.",
```

**Comment by EugeneHlushko** [comment]
Thanks!

**Comment by montogeek** [comment]
It looks kind of weird to be an official plugin but no maintainer.

**Comment by evilebottnawi** [comment]
@montogeek What do you mean? It is get updates almost every day

**Comment by montogeek** [comment]
@evilebottnawi Yep, I see that, but we didn't put it in the configuration :(

**Comment by evilebottnawi** [comment]
@montogeek we will put it by default after implement css out of box, don't worry

**Pull request merged**